doc/api/_toc.md

```diff
@@ -47,5 +47,6 @@
 
 <div class="line"></div>
 
+* <a href="#" data-js-handler="offline-docs">Make available offline</a>
```

Comment:
Would it make sense to add this link programmatically only if serviceWorkers are available? This would allow to avoid the `displayNoSwSupport` all the logic.